src/internal/deployment.js

```diff
@@ -117,11 +131,9 @@ class Deployment {
     }
 
     const deploymentId = this.deploymentInfo.id || this.buildVersion
-    const timeout = Number(core.getInput('timeout'))
     const reportingInterval = Number(core.getInput('reporting_interval'))
     const maxErrorCount = Number(core.getInput('error_count'))
```

Comment:
📝 We should add a follow-up issue to sanitize and validate these other numerical inputs as well.

Reply:
Logged as:
- #171 
- #172